.github/workflows/GitHubExt-CI.yml

```diff
@@ -0,0 +1,80 @@
+#
+# This workflow will build and run all unit tests.
+#
+
+name: DevHome-CI

```

Comment:
Did you want to keep the name the same as the one from DevHome or reflect the name of this yml.

Reply:
Fixed.